cmake/aws-c-io-config.cmake

```diff
@@ -1,6 +1,7 @@
 include(CMakeFindDependencyMacro)
 
 if (UNIX AND NOT APPLE)
+    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/modules")
```

Comment:
Should this be `list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/modules")`?

Reply:
That looks better, will do